pkgs/tools/text/termbook/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "termbook-cli";
+  version = "1.4.6";
+
+  src = fetchFromGitHub {
+    owner = "Byron";
+    repo = "termbook";
+    rev = "v${version}";
+    sha256 = "Bo3DI0cMXIfP7ZVr8MAW/Tmv+4mEJBIQyLvRfVBDG8c=";
+  };
+
+  cargoSha256 = "sha256-9fFvJJlDzBmbI7hes/wfjAk1Cl2H55T5n8HLnUmDw/c=";
+
+  meta = with lib; {
+    description = "A runner for `mdbooks` to keep your documentation tested.";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ phaer ];
+    homepage = "https://github.com/Byron/termbook/";
```

this is nice for tooling (e.g. [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) and [nix-update](https://github.com/Mic92/nix-update)) to generate more useful messages, also it would be nice if these were moved up, right under `description`

```suggestion
    homepage = "https://github.com/Byron/termbook/";
    changelog = "https://github.com/Byron/termbook/blob/${src.rev}/CHANGELOG.md";
```